src/earn/poolInfoScreen/BeforeDepositBottomSheet.test.tsx

```diff
@@ -17,139 +19,99 @@ const mockToken = {
   lastKnownPriceUsd: new BigNumber(1),
 }
 
-describe('BeforeDepositBottomSheet', () => {
-  it('show bottom sheet correctly when hasDepositToken is true, no other tokens', () => {
-    const { getByTestId } = render(
-      <Provider store={createMockStore()}>
-        <BeforeDepositBottomSheet
-          forwardedRef={{ current: null }}
-          token={mockToken}
-          pool={mockEarnPositions[0]}
-          hasDepositToken={true}
-          hasTokensOnSameNetwork={false}
-          hasTokensOnOtherNetworks={false}
-          canAdd={true}
-          exchanges={[]}
-        />
-      </Provider>
-    )
-    expect(getByTestId('Earn/ActionCard/Deposit')).toBeTruthy()
-    expect(getByTestId('Earn/ActionCard/AddMore')).toBeTruthy()
```

Comment:
these tests were only asserting for presence, which means if the code returns all actions for all options, all tests will pass 😅